samples/readme.md

```diff
@@ -40,7 +45,11 @@ The tests that are added to `/samples/unit-tests` are also part of the
 pre-commit tests that run via karma. Run `gulp test` on the root to pre-check.
 Read more at [highcharts/test](https://github.com/highcharts/highcharts/tree/master/test).
 
-### Useful tips for setting up tests
+
+
+Useful Tips for setting up Tests
```

Comment:
Use either all caps or no caps, but consistently

Reply:
https://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html